### PR TITLE
refactor: Convert function names to camelCase

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

**Review by lipis — APPROVED** [review]

File: src/script/conversation/ConversationRepository.ts [hunk 1]
```diff
@@ -1504,11 +1504,11 @@ export class ConversationRepository {
    * @param conversationEntity Conversation entity to delete
    * @param timestamp Optional timestamps for which messages to remove
    */
-  private _clear_conversation(conversationEntity: Conversation, timestamp?: number) {
+  private _clearConversation(conversationEntity: Conversation, timestamp?: number) {
```
Comment (lipis) [inline src/script/conversation/ConversationRepository.ts:1507]: I guess not part of this PR.. but I think we stopped prepending `_` to private ones as it's clear now.

Comment (ffflorian) [reply src/script/conversation/ConversationRepository.ts:1507]: Yeah I know. I will do that in another PR :)

**Review by Yserz — APPROVED** [review]

**Comment by bennycode** [comment]
Outcome of this refactoring:

```
- 1 X Content Search - TypeError: Unable to process binding "attr: function(){return…essage.display_edited_timestamp is not a function
- 11 X Direct Replies - Unable to locate element: {"method":"css selector","selector":"[data-uie-name='message-edited-reply-box']"}
- 12 X Edit - Expected condition failed: waiting for visibility of text '[edited]' in elements [] located by By.cssSelector: [data-uie-name='item-message']
- 1 X Legal Hold - Uncaught TypeError: Unable to process binding "attr: function(){return…essage.display_edited_timestamp is not a function
- 5 X Like - Expected condition failed: waiting for visibility of text '[edited message]' in elements [] located by By.cssSelector: [data-uie-name='item-message']
java.lang.AssertionError: Likes of others are visible for last message
- 1 X Link Preview - Uncaught TypeError: Unable to process binding "attr: function(){return…essage.display_edited_timestamp is not a function
```

We have to be really careful when converting names because the IDE's refactoring functionality might not catch names within Knockout templates. ⚠
